index.html

```diff
@@ -0,0 +1,116 @@
+<html lang="en">
+<head>
+    <head>
+        <link rel="stylesheet" href="style.css">
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Thinus Capstone</title>
+    </head>
+    <body class="BGColor">
+        <!-- col-1 will be used instead of col-xs-1 since bootstrap 5 no longer supports xs screen size -->
+        <nav class="navbar-fluid">
+            <div class="MobileMenu hidden">
+                <ul class="MobileNavBottom row col-12 col-sm-12">
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="pages/about.html">About</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="index.html">Program</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">Join</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">Sponsor</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">News</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">Warframe Lore</a></li>
+                </ul>
+            </div>
+            <div class="NavTop container-fluid">
+                <ul class="row col-lg-4 col-4 col-sm-4 offset-sm-8">
+                    <li class="Socials col-lg-1 col-1 col-sm-1"><a href="index.html"><img class="socials" src="images/fbicon.png" alt="Facebook Icon"></a></li>
+                    <li class="Socials col-lg-1 col-1 col-sm-1"><a href="index.html"><img class="socials" src="images/twittericon.png" alt="Twitter Icon"></a></li>
+                    <li class="NavText col-lg-2 col-2 col-sm-2">English</li>
+                    <li class="NavText col-lg-2 col-2 col-sm-2">My Page</li>
+                    <li class="NavText col-lg-2 col-2 col-sm-2">Logout</li>
+                </ul>
+            </div>
+            <div class="container-fluid">
+                <ul class="NavBottom row col-lg-11 col-11 col-sm-11 offset-1">
+                    <li class="col-lg-1 col-1 col-sm-1"><a href="index.html"><img class="col-lg-5 offset-lg-11 col-5 col-sm-5" src="images/lotusIcon.jpg" alt="lotusIcon"></a></li>
+                    <li class="Bnav AboutColor col-lg-1 col-1 col-sm-1 offset-3"><a href="pages/about.html">About</a></li>
+                    <li class="Bnav Color col-lg-1 col-1 col-sm-1"><a href="index.html">Program</a></li>
+                    <li class="Bnav Color col-lg-1 col-1 col-sm-1"><a href="#">Join</a></li>
+                    <li class="Bnav Color col-lg-1 col-1 col-sm-1"><a href="#">Sponsor</a></li>
+                    <li class="Bnav Color col-lg-1 col-1 col-sm-1"><a href="#">News</a></li>
+                    <li class="Bnav Color Lore col-lg-2 col-2 col-sm-2"><a href="#">Warframe Lore</a></li>
+                </ul>
```

Comment:
- [ ] Kindly note that the bottom navigation has a white background and the last link to the right has an orange border around it. Please implement this in your design so as to match the given design as much as possible

Given design
![image](https://user-images.githubusercontent.com/81932478/205433418-23d827f5-1150-4a57-83b4-24e1b70bf419.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/205433429-9e23a079-bd4c-4e75-825b-8d92eb1238d5.png)